### DISCUSSION TITLE
how to use ambient waypoint with destinationRule

### DISCUSSION THREAD

**Discussion by shuxnhs** [istio/istio#53672] [General]
can i use destinationRule'subset on httproute's backendRefs. who can give me a example

**Comment by ilrudie** [comment]
No. You can use the Kubernetes Gateway API style of traffic splitting though with a service per subset.

**Reply by shuxnhs** [reply]
I saw the update announcement for 1.23： Support for DestinationRule in waypoint proxies.  
What does that mean？

https://istio.io/latest/news/releases/1.23.x/announcing-1.23/

**Reply by howardjohn** [reply]
You can use everything in destination rule. However, HTTPRoute doesn't have a way to reference a subset
